src/WebJobs.Script/OutOfProc/Http/MessageExtensions/HttpMessageConversionExtensions.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script.OopHttp
+{
+    internal static class HttpMessageConversionExtensions
+    {
+        private static readonly JsonSerializerSettings _datetimeSerializerSettings = new JsonSerializerSettings { DateParseHandling = DateParseHandling.None };
+
+        public static Task<object> ToObject(this HttpResponseMessage responseMessage)
```

Comment:
This is essentially exposing a new API for the same functionality supported by the `HttpMessageExtensionUtilities`. Let's either use that from the caller or move the functionality in `HttpMessageExtensionUtilities` to `HttpResponseExtensions`

Reply:
yes. I still have to clean up the converters.

Reply:
deleted these. Will add converters in a separate PR.